lib/companies/schema/company.ex

```diff
@@ -13,7 +13,10 @@ defmodule Companies.Schema.Company do
     field :url, :string
 
     belongs_to :industry, Industry
-    has_many :jobs, Job
+    has_many :jobs, Job, defaults: [is_deleted: false]
+
+    field :is_deleted, :boolean
```

Comment:
For all of the schemas can we group the fields together and the associations together? I think that's more often how it's seen and what folks will be most familiar with coming into the code.